doc/cmake/BuildDocumentation.cmake

```diff
@@ -74,9 +84,10 @@ if(DOCS_TESTING)
 endif()
 
 execute_process(
-  COMMAND ${XELATEX} -interaction=${THIS_TEX_INTERACTION} ${XELATEX_MEM_FLAGS} ${INNAME}.tex
+  COMMAND "${XELATEX}" --interaction=${THIS_TEX_INTERACTION} ${XELATEX_MEM_FLAGS} ${INNAME}.tex
```

Comment:
I know xelatex's help shows everything with a single leading `-` but it does understand `--` so I'm using that. 

cf https://tex.stackexchange.com/questions/423161/why-do-both-single-and-double-dash-engine-arguments-work